src/OrchardCore/OrchardCore.Localization.Abstractions/CultureAspect.cs

```diff
@@ -0,0 +1,23 @@
+using System.Globalization;
+
+namespace OrchardCore.Localization
+{
+    /// <summary>
+    /// Represents a culture metadata for the content item.
+    /// </summary>
+    public class CultureAspect
+    {
+        /// <summary>
+        /// Gets or sets the culture.
+        /// </summary>
+        public CultureInfo Culture { get; set; }
```

Comment:
How do you ensure that `CultureInfo` is never null?

Reply:
We can use a default value